src/components/templates/SiteNavbar/SiteNavbar.tsx

```diff
@@ -1,15 +1,20 @@
-// import { CgBell, CgProfile } from 'solid-icons/cg';
-// import { Show, splitProps } from 'solid-js';
+import { CgBell, CgProfile } from 'solid-icons/cg';
+import { createSignal, Show } from 'solid-js';
 import { A } from 'solid-start';
-import { createServerData$ } from 'solid-start/server';
+// import { createServerData$ } from 'solid-start/server';
```

Comment:
Remove commented out code. If needed it can always be found in the git history (can make a commit that adds and then another to remove if necessary)

Reply:
How is that better? In this case, I intend to actually implement auth very soon and I don't see a good reason to delete code that I will uncomment as soon as I have a working form

Reply:
Okay